packages/bot-web-ui/src/components/trade-animation/__test__/trade-animation.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { act, render, screen } from '@testing-library/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import userEvent from '@testing-library/user-event';
+import { contract_stages } from 'Constants/contract-stage';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/root-store';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import TradeAnimation from '../trade-animation';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+    stopBot: jest.fn(),
+    getStrategySounds: jest.fn(() => []),
+    shouldRunBot: jest.fn(() => true),
+    runBot: jest.fn(),
+}));
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
+
+jest.useFakeTimers();
+
+window.Blockly = {
+    derivWorkspace: {
+        getAllBlocks: jest.fn(() => ({
+            find: jest.fn(),
+        })),
+        find: jest.fn(),
+    },
+};
```

Comment:
Tests are passing but with warnings so I mocked blocky and the methods giving warning. Since the only objective was to test the `TradeAnimation` block @maryia-matskevich-deriv 

<img width="1312" alt="Screenshot 2024-02-26 at 5 38 25 PM" src="https://github.com/binary-com/deriv-app/assets/90243468/cf1aeaaf-e29e-4719-85f8-afc858f5f008">